platform/linux-generic/odp_crypto_openssl.c

```diff
@@ -621,7 +621,8 @@ odp_crypto_alg_err_t auth_cmac_check(odp_packet_t pkt,
 static
 int internal_aad(EVP_CIPHER_CTX *ctx,
 		 odp_packet_t pkt,
-		 const odp_crypto_packet_op_param_t *param)
+		 const odp_crypto_packet_op_param_t *param,
+		 odp_bool_t gen)
```

Comment:
Based on the following function calls (`EVP_EncryptUpdate()`, `EVP_DecryptUpdate()`) `encrypt` (or `decrypt`) would in my opinion be a more descriptive variable name. Some specific reason to use `gen` here?

Reply:
This was related to `gen`/`check` phases. I'm changing that to `encrypt`/`decrypt` correspondingly.